### ISSUE TITLE
Select from PDF background

### ISSUE THREAD

**Issue by Yaniv-Kaplan** [mb1986/rm-hacks#215]
As a student I often find myself reading a question in a PDF, then writing the answer to said question in a seperate Notebook.

This either requires that I swap a lot between the documents, or that I manually copy the question to the notebook. Both aren't ideal.

**Suggestions**:
- Somehow allow copying from the background layer of the PDF (1). 
- Somehow allow adding images into notebooks (2).

(1) I once tried to do this by converting the PDF into vectors, then converting them into rM's "Lines" format (in other words, converting the PDF into a regualr notebook, with the background layer being registered as regular handwriting). It almost worked, but I concluded that so long as it isn't integrated into the PDF, it won't really help.

(2) Will probably never happen without support form rM themselves.

**Comment by atngames** [comment]
Would a split screen capacity help you with this issue ?

**Comment by atngames** [comment]
https://github.com/mb1986/rm-hacks/issues/216

**Comment by Yaniv-Kaplan** [comment]
> Would a split screen capacity help you with this issue ?

Definitely! But still this can be really useful in many other situations where the aplit screen will not help.

For example:
If I want to copy a sketch from the internet into my notebook to draw over or edit.
